data/oci-migrate-conf.yaml

```diff
@@ -0,0 +1,272 @@
+# ----------------------------------------------------------------------------
+# Configuration and status data with respect to the oci-image-migrate.
+# ----------------------------------------------------------------------------
+#
+# dummy format key
+dummy_format_key : '01234567'
+#
+# python version
+pythonver : 2
+#
+# log file path, without .log extension, timestamp and extension added runtime.
+# logfilepath : /var/tmp/oci-image-migrate
+#
+# result file path without extension, image references are added runtime,
+resultfilepath : /var/tmp/oci-image-migrate
+#
+# Dictionary containing utilities which might be needed with the packages
+# which provide them.
+helpers_list : 
+    qemu-nbd   : qemu-img
+    qemu-img   : qemu-img
+    blkid      : util-linux
+    lsblk      : util-linux
+    sfdisk     : util-linux
+    fdisk      : util-linux
+    parted     : parted,
+    partx      : util-linux
+    lvmdiskscan: lvm2
+    oci        : oci-cli
+#
+# logging configuration
+logging_config:
```

Comment:
Why are we defining again logging ? is it not based on logging we already have in oci_utils ?